modules/services/mpd-discord-rpc.nix

```
@@ -0,0 +1,50 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.mpd-discord-rpc;
+  tomlFormat = pkgs.formats.toml { };
+  configFile = tomlFormat.generate "config.toml" cfg.settings;
+
+in {
+  meta.maintainers = [ maintainers.kranzes ];
+
+  options.services.mpd-discord-rpc = {
+    enable = mkEnableOption "Whether to enable the mpd-discord-rpc service";
```

```suggestion
    enable = mkEnableOption "the mpd-discord-rpc service";
```

`mkEnableOption` already adds the "Whether to enable" text